### PR TITLE
Updated UI

### PR DESCRIPTION
**Pull request by tjncgit**
UI is more clean and organized

### CODE REVIEW

File: project/src/main/webapp/catalog.js [hunk 1]
```diff
@@ -1,75 +1,70 @@
...
-        const request = {
-            placeId: placeID,
-            fields: ["photos", "rating", "reviews"]
-        };
+    removeAllChildNodes(reviewArea);
+    containerThree = document.getElementById("container-3").removeAttribute("hidden");
+    messageContainer = document.getElementById("chat-div").removeAttribute("hidden");
+    const request = {
+        placeId: placeID,
+        fields: ["photos", "rating", "reviews", "name", "formatted_address"]
```
Comment (Howon) [inline project/src/main/webapp/catalog.js:14]: why don't you delcare it outside this funciton scope if this is not going to change?

File: project/src/main/webapp/catalog.js [hunk 2]
```diff
@@ -1,75 +1,70 @@
...
-                if (!photos) {
-                    console.log("No Photos");
-                    return;
-                }
+    service = new google.maps.places.PlacesService(map);
+    service.getDetails(request, function callback(results, status) { // this request will return an array of photos as a result
+        if (status == google.maps.places.PlacesServiceStatus.OK) {
+            photos = results.photos;
+            reviews = results.reviews;
+            console.log(results);
```
Comment (Howon) [inline project/src/main/webapp/catalog.js]: Please fix indentation.

File: project/src/main/webapp/catalog.js [hunk 3]
```diff
@@ -1,75 +1,70 @@
...
+            if (!photos) {
+                console.log("No Photos");
```
Comment (Howon) [inline project/src/main/webapp/catalog.js]: Delete

**Review by Howon — CHANGES_REQUESTED** [review]

File: project/src/main/webapp/catalog.js (hunk 2 above)
Comment (tjncgit) [reply project/src/main/webapp/catalog.js]: whats wrong with the indentation

File: project/src/main/webapp/catalog.js (hunk 1 above)
Comment (tjncgit) [reply project/src/main/webapp/catalog.js:14]: request variable is used in other js files so it would cause an error if i declare outside of the function

**Pull request merged**